resources/lib/Zype/Api.php

```diff
@@ -1,111 +1,47 @@
 <?php
 
-use GuzzleHttp\Client;
-use GuzzleHttp\HandlerStack;
-use League\Flysystem\Adapter\Local;
-use Kevinrob\GuzzleCache\CacheMiddleware;
-use Kevinrob\GuzzleCache\Strategy\GreedyCacheStrategy;
-use Kevinrob\GuzzleCache\Storage\FlysystemStorage;
-
-class Api {
-    /**
-     * Http client.
-     *
-     * @var \GuzzleHttp\Client
-     */
+class Api
+{
+
     private static $authClient;
     private static $resourceClient;
     private static $resourceClientHttp;
-    private static $resourceClientCached;
     private static $options;
+    private static $body;
 
     /**
      * Api options.
      *
      * @var array
      */
-    public function __construct($options = array()) {
-        try {
-            self::$options = $options;
-
-            $authPoint = trim(self::$options['authpoint'], '/') . '/';
-            $endPoint = trim(self::$options['endpoint'], '/') . '/';
+    public function __construct($options = array())
+    {
+        self::$options = $options;
 
-            self::$authClient = new Client([
-                'base_uri' => $authPoint,
-                'timeout' => 120,
-            ]);
+        $authPoint = trim(self::$options['authpoint'], '/') . '/';
+        $endPoint = trim(self::$options['endpoint'], '/') . '/';
 
-            self::$resourceClient = new Client([
-                'base_uri' => $endPoint,
-                'timeout' => 120,
-            ]);
-
-            self::$resourceClientHttp = new Client([
-                'base_uri' => str_replace('https://', 'http://', $endPoint),
-                'timeout' => 120,
-            ]);
-
-            $cache_dir = new Local(__DIR__ . '/cache');
-
-            if (defined('WP_CONTENT_DIR')) {
-                $upload_dir = wp_upload_dir();
-                $cache_dir = $upload_dir['basedir'] . '/zype';
-
-                if (!file_exists($cache_dir)) {
-                    wp_mkdir_p($cache_dir);
-                }
-
-                $cache_dir = new Local($cache_dir);
-            }
+        self::$authClient = $authPoint;
+        self::$resourceClient = $endPoint;
```

Comment:
@Osoro `authClient` and `resourceClient`, aren't they misnamed?  They aren't actually a client, are they?